### PR TITLE
Create sunn4github.json

### PR DESCRIPTION
**Pull request by sunn4room**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

[The link](https://ghproxy.sunn4room.workers.dev)

### CODE REVIEW

File: domains/sunn4github.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "sunn4room",
+        "email": "sunn4room@163.com"
+    },
+    "record": {
+        "CNAME": "ghproxy.sunn4room.workers.dev"
```
Comment (DEV-DIBSTER) [inline domains/sunn4github.json:7]: Little content provided.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/sunn4github.json (hunk 1 above)
Comment (sunn4room) [reply domains/sunn4github.json:7]: Completed !  [The Link](https://ghproxy.sunn4room.workers.dev)

Comment (DEV-DIBSTER) [reply domains/sunn4github.json:7]: This just looks like an exact copy of GitHub homepage?

Comment (sunn4room) [reply domains/sunn4github.json:7]: Yes, it's a cf worker to access github. It's difficult to access github in China due to DNS attack. T.T ~~~

Comment (DEV-DIBSTER) [reply domains/sunn4github.json:7]: Ah got it.

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**